Flowchart/Views/Home/Contact.cshtml

```diff
@@ -2,4 +2,18 @@
     ViewData["Title"] = "Contact";
 }
 <h2>@ViewData["Title"].</h2>
+<ul>
+    <li>
+        <a href="mailto:#">samuel.messina@siu.edu</a>
```

Comment:
@reidtevonian These links are broken. You could also make the text something like "Email Sam" instead of the email directly for better UX. 

Check out the example on the mailto link [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_mailto)